src/ui/components/Collections/Collections.jsx

```diff
@@ -1,81 +1,215 @@
-import React from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
 import { useQuery } from 'react-query';
 import {
   Table,
   TableBody,
   TableCell,
   TableContainer,
-  TableHead,
-  TableRow
+  TableRow,
+  TablePagination
 } from '@material-ui/core';
 import ReactJson from 'react-json-view';
 import PropTypes from 'prop-types';
+import useStyles from './styles';
+import SortedTableHead from './SortedTableHead';
+import DeleteIcon from '@material-ui/icons/Delete';
+import CreateIcon from '@material-ui/icons/Create';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/Add';
 
 const Collections = props => {
+  const classes = useStyles();
   const { selectedCollection } = props;
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [order, setOrder] = useState('asc');
+  const [orderBy, setOrderBy] = useState('');
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  function descendingComparator(a, b, orderBy) {
+    if (b[orderBy] < a[orderBy]) {
+      return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+      return 1;
+    }
+    return 0;
+  }
+
+  function serverStatus(server) {
+    return server && true;
+  }
+
+  function getComparator(order, orderBy) {
+    return order === 'desc'
+      ? (a, b) => descendingComparator(a, b, orderBy.toLowerCase())
```

Comment:
This might be a non-issue, but I'm concerned about doing a `toLowerCase` here -- is it possible we'll have fields with camelCase names? To be extra defensive I'd suggest we use the "official" name in the definition below and then just render it with all-caps.